ci/storage/s3.go

```diff
@@ -139,9 +148,9 @@ func (s *Storage) GetCacheableFile(bucket string, predicate func(s3.Object) bool
 		}
 		defer file.Close()
 
-		downloader := s3manager.NewDownloaderWithClient(s)
+		downloader := manager.NewDownloader(s)
 
-		numBytes, err := downloader.Download(file, &s3.GetObjectInput{
+		numBytes, err := downloader.Download(context.TODO(), file, &s3.GetObjectInput{
```

Comment:
As we now have context here, we could add a timeout, though I was not sure what is the amount of time we can wait for the file download, so I just added `TODO()`. If anyone thinks it's worth to timeout here, you're free to make suggestions.